File: c++/XString38.cpp
#include "pt4.h"
using namespace std;

void Solve()
{
    Task("XString38");
	string s,s1,s2;
	pt >> s;
	pt >> s1; pt >> s2;
	int pos = s.find(s1);
	while (pos != -1){
        string tmp = s;
        string tmp_beg = tmp.substr(0,pos);
        string tmp_end = tmp.substr(pos+s1.length(),s.length()-s1.length()-pos);
        s.erase();
        s += tmp_beg + s2 + tmp_end;
        pos = s.find(s1);
    }
	pt << s;
}

File: c#/XString38.cs
// File: "XString38"
using PT4;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PT4Tasks
{
    public class MyTask : PT
    {
        public static void Solve()
        {
            Task("XString38");
            var s =GetString();
            var s1 = GetString();
            var s2 = GetString();
            var ans = s.Replace(s1,s2);
            Put(ans);
        }
    }
}
